packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/new/new.html

```diff
@@ -0,0 +1,58 @@
+<div data-ovh-alert="app.dedicatedClouds.datacenter.backup.new"></div>
+<p data-translate="dedicatedCloud_datacenter_backup_new_description1"></p>
+<p
+    data-ng-bind-html="::'dedicatedCloud_datacenter_backup_new_description2' | translate"
+></p>
+<strong>
+    <a
+        target="_blank"
+        rel="noopener"
+        class="oui-link_icon"
+        data-ng-href="{{ ::$ctrl.backupTariffUrl }}"
+    >
+        <span
+            data-translate="dedicatedCloud_datacenter_backup_new_tariff_url"
+        ></span>
+        <span
+            class="oui-icon oui-icon-arrow-right_extra-thin"
+            aria-hidden="true"
+        ></span>
+    </a>
+</strong>
+<oui-stepper class="mt-3" data-on-finish="$ctrl.orderBackup()" data-current-index="$ctrl.data.currentStep">
+    <oui-step-form
+        data-header="{{ 'dedicatedCloud_datacenter_backup_new_choose_offer' | translate }}"
```

Comment:
```suggestion
        data-header="{{:: 'dedicatedCloud_datacenter_backup_new_choose_offer' | translate }}"
```
We should have this everywhere  when it's required

Reply:
This is done in all places where required :)